spec/factories/order.rb

```diff
@@ -0,0 +1,13 @@
+FactoryBot.define do
+  factory :order do
+    guests { 10 }
+    delivery { "A domicilio" }
```

Comment:
please use single quotes for ruby code when you can (except on string interpolation)